#coefficients for detected planes
Header header
single_plane[] Coeff